doc/ug/getting_started_verilator.md

```diff
@@ -52,7 +52,7 @@ $ build/lowrisc_dv_chip_verilator_sim_0.1/sim-verilator/Vchip_sim_tb \
   --meminit=otp,build-bin/sw/device/otp_img/otp_img_sim_verilator.vmem
 ```
 
-To stop the simulation press CTRL-c.
+To stop the simulation press CTRL-C.
```

Comment:
We had explicitly a lower-case C to indicate that it's CTRL-c, not CTRL-SHIFT-c. But I guess you could argue either way.

Reply:
You're correct, here it can be phrased either way. But if it's ``CTRL+SHIFT+C``, it'll always be like that, I don't think capitalizing should assert the usage of ``Shift``, rather it should be explicit.